stable/aws-for-fluent-bit/README.md

```diff
@@ -96,15 +116,81 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `elasticsearch.replaceDots` | Enable or disable Replace_Dots  | On |
 | `elasticsearch.suppressTypeName` | OpenSearch 2.0 and above needs to have type option being removed by setting Suppress_Type_Name On  | |
 | `elasticsearch.extraOutputs` | Append extra outputs with value | `""` |
-| `additionalOutputs` | add outputs with value | `""` |
-| `priorityClassName` | Name of Priority Class to assign pods | |
-| `updateStrategy` | Optional update strategy | `type: RollingUpdate` |
-| `affinity` | Map of node/pod affinities | `{}` |
-| `env` | Optional List of pod environment variables for the pods | `[]` |
-| `tolerations` | Optional deployment tolerations | `[]` |
-| `nodeSelector` | Node labels for pod assignment | `{}` |
-| `annotations` | Optional pod annotations | `{}` |
-| `volumes` | Volumes for the pods, provide as a list of volume objects (see values.yaml) |  volumes for /var/log and /var/lib/docker/containers are present, along with a fluentbit config volume |
-| `volumeMounts` | Volume mounts for the pods, provided as a list of volumeMount objects (see values.yaml) | volumes for /var/log and /var/lib/docker/containers are mounted, along with a fluentbit config volume |
-| `dnsPolicy` | Optional dnsPolicy | `ClusterFirst` |
-| `hostNetwork` | If true, use hostNetwork | `false` |
+
+| `s3.enabled` | Whether this plugin should be enabled or not, [details](https://docs.fluentbit.io/manual/pipeline/outputs/s3) | `true`
+| `s3.match` | The log filter. | `"*"`
+| `s3.bucket` | S3 Bucket name. |
+| `s3.region` | The AWS region of your S3 bucket. | `"us-east-1"`
+| `s3.jsonDateKey` | Specify the name of the time key in the output record. To disable the time key just set the value to false. | `"date"`
+| `s3.jsonDateFormat` | Specify the format of the date. Supported formats are double, epoch, iso8601 (eg: 2018-05-30T09:39:52.000681Z) and java_sql_timestamp (eg: 2018-05-30 09:39:52.000681). | `"iso8601"`
+| `s3.totalFileSize` | Specifies the size of files in S3. Maximum size is 50G, minimim is 1M. | `"100M"`
+| `s3.uploadChunkSize` | The size of each 'part' for multipart uploads. Max: 50M | `50M`
```

Comment:
I still think this is too large- why did you choose this value as default which is 10x the plugin default?

Reply:
I changed it to 5M but missed it in doc. 
updated here as well now.